### PR TITLE
#3178 - Updates to Provincial Daily Disbursement Report Sent to Finance - CSV Sent By Email (Part 2)

### PR DESCRIPTION
**Pull request by lewischen-aot**
As of the current story, provincial daily disbursement report files, including both full time and part time disbursement values, will be sent to the ministry user via emails. It is optional but beneficial to rename "full-time-disbursement-receipts-file-integration" related implementations to "disbursement-receipts-file-integration", where both full time and part time disbursement values are being considered. As part of the story, the following changes are completed in this PR:

- Renamed "full-time-disbursement-receipts-file-integration" implementations to "disbursement-receipts-file-integration":
  - Created a DB migration to rename the `queue_name` column
  - Renamed codes and files related to "full-time-disbursement-receipts-file-integration" to "disbursement-receipts-file-integration", e.g., `FullTimeDisbursementReceiptsFileIntegrationScheduler` to `DisbursementReceiptsFileIntegrationScheduler`
  - Adjusted E2E test file `disbursement-receipts-integration.scheduler.e2e-spec.ts` and its read-in e-cert file.
- Removed the class `DailyDisbursementReceiptProcessingService` (and this file) and the method `getMaxDisbursementReceiptDate`

The next PR will have the following changes:
- Adjust `report_sql` for `Daily_Disbursement_File` in the `report_configs` table.
  - Consider only the BC funding (BC or BP)
- Implement the method `processProvincialDailyDisbursements` into class `DisbursementReceiptProcessingService`

### CODE REVIEW

**Comment by guru-aot** [comment]
Please remove the no longer referenced services and methods, as part of this story.
    class DailyDisbursementReceiptProcessingService
    method DisbursementReceiptService.getMaxDisbursementReceiptDate
As mentioned in the previous PR.

**Comment by guru-aot** [comment]
Please check on the new issue in the sonarcloud. There is a deprecated method for QueueProcessSummary. Please change it to process summary in your implementation.

File: sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts [hunk 1]
```diff
@@ -311,6 +314,99 @@ describe(
...
+        feReceipt.disbursementReceiptValues,
+      );
+      // Disbursed grants.
+      // XYZ is a non-translated code that must be imported as it is.
+      expect(feReceiptAwards).toStrictEqual({
+        CSGF: 199,
+        CSGT: 299,
+        CSGD: 399,
+        CSGP: 499,
+        XYZ: 444,
```
Comment (guru-aot) [inline sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts:384]: what is this XYZ? I cant understand can you please explain?

File: sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/disbursement-receipts-integration.scheduler.ts [hunk 2]
```diff
@@ -29,18 +29,15 @@ export class FullTimeDisbursementReceiptsFileIntegrationScheduler extends BaseSc
...
+    const processSummary = new ProcessSummary();
+    processSummary.info(
       `Processing full time disbursement receipts integration job ${job.id} of type ${job.name}.`,
     );
     const auditUser = this.systemUsersService.systemUser;
     const processResponse =
       await this.disbursementReceiptProcessingService.process(auditUser.id);
     await this.cleanSchedulerQueueHistory();
-    await summary.info(
+    processSummary.info(
```
Comment (guru-aot) [inline sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/disbursement-receipts-integration.scheduler.ts:40]: 👍

File: sources/packages/backend/libs/test-utils/src/factories/disbursement-receipt.ts [hunk 3]
```diff
@@ -96,7 +96,8 @@ export async function saveFakeDisbursementReceiptsFromDisbursementSchedule(
     disbursementSchedule.disbursementValues.find(
       (award) => award.valueType === DisbursementValueType.BCLoan,
     )?.valueAmount ?? 0;
-  provincialDisbursementReceipt.fundingType = RECEIPT_FUNDING_TYPE_PROVINCIAL;
+  provincialDisbursementReceipt.fundingType =
```
Comment (guru-aot) [inline sources/packages/backend/libs/test-utils/src/factories/disbursement-receipt.ts:99]: This means the default value of the test utils is fulltime right?

**Review by guru-aot — COMMENTED** [review]
Nice work @lewischen-aot . Added some more comments, please have a look

File: sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts (hunk 1 above)
Comment (lewischen-aot) [reply sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts:384]: XYZ is a non-translated code that must be imported as it is. This part of the code for the federal receipt is the same from previous test cases where the values for the federal receipt awards are acquired for `feReceiptAwards` via method `getExpectedAwardsFromReceiptValues` with input `feReceipt.disbursementReceiptValues`.

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @lewischen-aot

**Pull request merged**